util/blackjack-games.js

```diff
@@ -0,0 +1,134 @@
+const Deck = require('card-deck');
+const _ = require('lodash');
```

Comment:
nit: can we import just the clone function if thats all we're using from lodash? 
```
const clone = requre('lodash/clone')
```